### ISSUE TITLE
Add student startups to the site

### ISSUE THREAD

**Issue by johnwheal** [nestcommunity/site#4]
It would be great if all the student startups were listed on the site. Maybe with a link to their website.

**Comment by zoetrope69** [comment]
yeah i like this, could use flat files on the repo. might not be easy for everyone to add tho

**Comment by johnwheal** [comment]
It would ideally include logos and so I think adding it to the repo would be best. One of us could just add new startups.

**Comment by zoetrope69** [comment]
trying to decide whether Jekyll is overkill

**Comment by johnwheal** [comment]
I've never used it but it doesn't look overkill.

**Comment by zoetrope69** [comment]
works with github pages, here's an example: https://github.com/rosedigital/placeholster
